mmfunctions

```diff
@@ -6,6 +6,7 @@ OBJECTS_FIND_EXCLUSIONS=(! -name ".*")
 OBJECTS_FIND_EXCLUSIONS+=(! -path "*/access/*")
 OBJECTS_FIND_EXCLUSIONS+=(! -path "*/service/*")
 OBJECTS_FIND_EXCLUSIONS+=(! -path "*/trimmed_materials/*")
+OBJECTS_FIND_EXCLUSIONS+=(! -path "*/restoration/*")
```

Comment:
If this line is kept in, then it is unable to make access file from restoration because it does not recognize the input. Tested it without this line and then it worked. :)